DOM_creation/Lesson4/main.js

```diff
@@ -0,0 +1,18 @@
+const ul = document.getElementById("js-ul");
+const fragment = new DocumentFragment();
+const itemsToInsert = [{to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"}, {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}];
+
+for (let item of itemsToInsert ){
+  const li = document.createElement('li');
+  const a = document.createElement('a');
+  a.href = item.to;
+  a.textContent = item.text;
+
+  const img = document.createElement('img');
+  img.src = item.img;
+  img.alt = item.alt
```

Comment:
nits
```suggestion
  const li = document.createElement('li');
  const a = document.createElement('a');
  const img = document.createElement('img');
  a.href = item.to;
  a.textContent = item.text;
  img.src = item.img;
  img.alt = item.alt
```
It's a minor detail, but it's easier to read if it's separated this way.

Reply:
Hi Chihuyu-san,
Thank you for your suggestion!
I changed the order of codes inside of for of methods as per your suggestion.
Please check my latest commit b3dd714c16c38e5e03791e01c5837dd498a442b9